guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>Training is supported for features and features with Rasters.</p><p> </p><p>Probably may need to change this line.</p><p> </p><p>"It can take both feature data and raster data as input."</p><p> </p><p>Inferencing can happen on feature data, feature with Rasters, or just rasters.</p><p> </p><p>For FCN we could mention that it is based on TabularLearner from FastAI.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>